lib/output.ml

```diff
@@ -17,34 +17,45 @@
 open Misc
 open Compat
 
-type t = [ `Output of string | `Ellipsis ]
+type sub = S of string | Sub_ellipsis
+
+type t = [ `Output of sub list | `Ellipsis ]
```

Comment:
What would you think about having a third variant for raw lines so that we can use `String.equal` for those instead of going through a regexp?

Reply:
I added extra cases in the matches function instead